libpromises/dbm_migration.c

```diff
@@ -26,6 +26,7 @@
 
 #include <lastseen.h>
 #include <string_lib.h>
+#include <backup.h>
```

Comment:
A great generic name of the header file! :sweat_smile:  (out of scope of this PR, of course)